ETC.tsv

```diff
@@ -7799,24 +7799,24 @@ ETC_20150317_007799	Position saw in dream
 ETC_20150317_007800	Collect the antidote from people of Stone Pole Platform
 ETC_20150317_007801	Activate the sealed device
 ETC_20150317_007802	Check the royal device
-ETC_20150317_007803	The discharge seal release of Dzurura in earlier than Rexipher
-ETC_20150317_007804	Release a seal of Sviesa's altar
-ETC_20150317_007805	Ruklys of memorial (2)
-ETC_20150317_007806	Obtain a first tombstone part
-ETC_20150317_007807	Collect the fragment covered with dust
+ETC_20150317_007803 Unleash Gedulla's infantry seal before Rexipher 
+ETC_20150317_007804	Unseal Sviesa altar
+ETC_20150317_007805	Ruklys memorial(2)
+ETC_20150317_007806	Acquire the first tombstone piece
+ETC_20150317_007807	Collect a dusty ruin fragment
 ETC_20150317_007808	Ruklys re-evaluation (1)
 ETC_20150317_007809	Ruklys re-evaluation (2)
-ETC_20150317_007810	Sap a short tree
-ETC_20150317_007811	Obtain a part of buried remains
+ETC_20150317_007810	Tap a stubby tree
+ETC_20150317_007811	Obtain a buried remnant shard
 ETC_20150317_007812	Lydia Schaffen is leaving zeolite (2)
 ETC_20150317_007813	Collect the charcoal with Long arm tree
-ETC_20150317_007814	Third time rubbing work
-ETC_20150317_007815	Lydia Schaffen is leaving zeolite (4)
-ETC_20150317_007816	Lydia Schaffen is leaving zeolite (5)
+ETC_20150317_007814	The third rub 
+ETC_20150317_007815	Lydia Schaffen's tombstone (4)
+ETC_20150317_007816	Lydia Schaffen's tombstone (5)
 ETC_20150317_007817	Flurry's epitaph (7)
-ETC_20150317_007818	Request of solicitor Eedang
-ETC_20150317_007819	N/A
-ETC_20150317_007820	Open the front gate of Sanctum (4)
+ETC_20150317_007818	Solicitor Edam's Request
+ETC_20150317_007819 Tower of Mage 42 interactive session
```

Comment:
IMC has mentioned that lines with **N/A** should not be translated. Furthermore, it should be Mage Tower.